src/openloco/companymgr.cpp

```diff
@@ -113,4 +119,76 @@ namespace openloco::companymgr
             }
         }
     }
+
+    // 0x004383ED
+    void updateOwnerStatus()
+    {
+        if (openloco::is_title_mode() || openloco::is_editor_mode())
+        {
+            return;
+        }
+
+        auto company = companymgr::get(_updating_company_id);
+        if (company == nullptr)
+        {
+            return;
+        }
+
+        company->var_12 += 1;
+        if ((company->var_12 & 0x7F) != 0)
+            return;
+
+        for (size_t i = 0; i < WindowManager::count(); i++)
+        {
+            auto w = WindowManager::get(i);
+
+            if (w->type != WindowType::vehicle)
+                continue;
+
+            auto vehicle = thingmgr::get<openloco::vehicle>(w->number);
+            if (vehicle->x == -0x8000)
+                continue;
+
+            if (vehicle->owner != _updating_company_id)
+                continue;
+
+            game_commands::do_73(vehicle->id);
```

Comment:
We can be pretty confident that CS uses game commands in the same way as RCT2 should we maybe pass bl the flags variable into the `game_commands` functions? I'm not sure if this was already discussed.

Reply:
I'm 99% sure this game command is only used in this function. It seemed unnecessary to pass arguments that are the same for all calls. Although it hasn't been discussed.